README.md

```diff
@@ -4,39 +4,53 @@
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/devconsole-operator/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/devconsole-operator)
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/operator-registry/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/operator-registry)
 
-DevConsole operator enables a developer-focused view in OpenShift 4.
-It provides a view switcher to transition between the traditional
-Kubernetes cluster administration console referred to as
-Administrator, to this new Developer perspective.
+## Overview
+The DevConsole operator enables a developer-focused view in the OpenShift 4 web console.
+It provides a view switcher to transition between the traditional Kubernetes cluster administration console referred to as Administrator, to a new Developer perspective.
 
-This new Developer perspective provides a high-level abstraction over
-Kubernetes and OpenShift primitives to allow developers to focus on
-their application development.
+This new Developer perspective provides a high-level of abstraction over Kubernetes and OpenShift primitives to allow developers to focus on their application development.
 
 ## Key Features
 
-The Developer perspective is still under active development.  These
-are the main features that are getting developed:
+The Developer perspective is under active development. These
+are the main features being developed:
 
-* Add - The place to create and build the application using one of this method:
+* **Add**: Use this page to create and build an application using one of the following methods:
 
-	- Importing source code from Git
-	- Deploying an existing image
-	- Browse a catalog to deploy or connect application services
-	- Deploy quick-starters or samples
+    - Import source code from Git
+    - Deploy an existing image
+    - Browse the catalog to deploy or connect application services
+    - Deploy quick-starts or samples
 
-* Topology - The landing page that shows application structure and
-  health in an easy-to-use diagram
-* Builds - Lists OpenShift BuildConfig resources for the selected
-  project
-* Pipelines - Lists Tekton Pipeline resources for the selected project
+* **Topology**: The landing page that shows the application structure and
+  health in an easy-to-use graphic representation.
+* **Builds**: This page lists the OpenShift BuildConfig resources for the selected project.
+* **Pipelines**: This page lists the Tekton Pipeline resources for the selected project.
+
+## Installing the latest console with the developer perspective as a non-admin user
+
+To install the latest console with the developer perspective:
+
+1. Clone the [devconsole repository](https://github.com/redhat-developer/devconsole-operator) locally.
+1. Change directory to the `hack/install_devconsole` directory and run the script:
+```
+sh consoledeveloper.sh
+```
+
+    The script:
```

Comment:
Indentation is not correct. See the preview.

Reply:
Thanks for catching this Baiju, I also saw indentation issues with some of the numbered steps in the sections below. Fixed them too. Strange that Atom editor does not catch this. I think it is more suited for asciidoc. Gist worked well in catching this.

Reply:
@baijum please check now.